java/api/TypeDBSession.java

```diff
@@ -144,15 +140,6 @@ enum Type {
             this.isSchema = nativeObject == com.vaticle.typedb.driver.jni.SessionType.Schema;
         }
 
-        public static Type of(com.vaticle.typedb.driver.jni.SessionType sessionType) {
```

Comment:
We decided that it's not needed for our code and users actually don't need (or even have not to) to use native values to get Java's objects, so we removed it.